tests/js/client/chaos/test-chaos-load-common.inc

```diff
@@ -222,12 +218,12 @@ function BaseChaosSuite(testOpts) {
             console.info(msg);
           }
         };
-        const ops = isTransaction ? Math.floor(Math.random() * 5) + 1 : 1;
+        const ops = trx === 0 ? 1 : Math.floor(Math.random() * 5) + 1;
```

Comment:
It doesn't look like `trx` is ever `0`, but rather `null` or an object.

Reply:
🤦‍♂️